hip/static/js/index.js

```diff
@@ -25,5 +25,20 @@ import RightScrollNav from "../../../apps/hip/static/js/rightScrollNav";
 document.addEventListener("DOMContentLoaded", function() {
   Header();
   SideBar();
-  RightScrollNav();
+  lazyLoad();
+
+  function lazyLoad() {
+    return lazyLoadRightScrollNav()
+  };
+  function lazyLoadRightScrollNav() {
+    /* Lazy Load Right Scroll Nav Component Javascript
+    This javascript should only run on pages that use the right scroll nav component 
+    */
+    const rightScrollNavPages = ["report-a-disease"];
```

Comment:
Wonder if it would be possible to look for the `right-scroll-container-hip` div instead? That way we wouldn't have to keep this list of pages updated.

Reply:
I will try, I don't think `right-scroll-container-hip` is in the DOM on each page. I will need to find out the best method for checking that the element exists without throwing a reference error. Or may be we could use a try/catch block for this.

Reply:
Exactly. That div is only on the pages where we want the JS to load, so it is the perfect thing to be our conditional check. Let me know if you want me to work on this, I'm sure I've done it before... just need to remember how :)

Reply:
done. Vastly simplified.